src/plugins/lua/known_senders.lua

```diff
@@ -53,6 +53,11 @@ local settings = {
   symbol = 'KNOWN_SENDER',
   symbol_unknown = 'UNKNOWN_SENDER',
   redis_key = 'rs_known_senders',
+  sender_prefix = 'rsrk',
+  rsrk_privacy = false,
+  rsrk_privacy_alg = 'blake2',
+  rsrk_privacy_prefix = 'obf',
+  rsrk_privacy_length = 16,
 }
```

Comment:
I don't like the idea of prefixing these settings with `rsrk` - it looks obscure. Maybe it is better to get the meaningful prefix, such as `reply_sender_privacy` for these options?